Doc/library/typing.rst

```diff
@@ -1616,7 +1616,7 @@ without the dedicated syntax, as documented below.
 
 .. _typevar:
 
-.. class:: TypeVar(name, *constraints, bound=None, covariant=False, contravariant=False, infer_variance=False)
+.. class:: TypeVar(name, *constraints, bound=None, covariant=False, contravariant=False, infer_variance=False, default=None)
```

Comment:
I don't think the signature of `TypeVar` itself should have `default=None`, because otherwise it's impossible to distinguish a TypeVar without a default from one that actually defaults to having type `None` (https://github.com/python/typing_extensions/pull/85)

Reply:
Good point, that's actually currently incorrect in the runtime code: `TypeVar("T")` and `TypeVar("T", default=None)` cannot be distinguished. Do you know how we should present this in the docs? Maybe this:
```suggestion
.. class:: TypeVar(name, *constraints, bound=None, covariant=False, contravariant=False, infer_variance=False, default=<marker>)
```

Reply:
not sure. Some other options:

```suggestion
.. class:: TypeVar(name, *constraints, bound=None, covariant=False, contravariant=False, infer_variance=False, default=<unrepresentable>)
```

"overload-style" -- this is already done elsewhere, e.g. for the constructor at https://docs.python.org/3/library/stdtypes.html#str -- but annoyingly verbose here...

```suggestion
.. class:: TypeVar(name, *constraints, bound=None, covariant=False, contravariant=False, infer_variance=False)
           TypeVar(name, *constraints, default, bound=None, covariant=False, contravariant=False, infer_variance=False)
```

"man-page" style (does it make sense to mix "man-page" syntax with the Python syntax for defaults that the following parameters use?):

```suggestion
.. class:: TypeVar(name, *constraints[, default], bound=None, covariant=False, contravariant=False, infer_variance=False)
```

Reply:
Thanks, I like the "<unrepresentable>" variant best. The overload style is too verbose and mixing the bracket style and normal defaults looks weird.